src/styles/Home.css

```diff
@@ -0,0 +1,50 @@
+.gainers-container {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  width: 100%;
+}
+
+.detail-link,
+.gainers-search {
+  display: flex;
+  flex-direction: column;
+  padding: 0.5rem;
+}
+
+.gainers-search {
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 0.5rem;
+  gap: 0.5rem;
+}
+
+.form {
+  padding-top: 10px;
+  width: 100%;
+  max-width: 400px;
+  display: flex;
+  align-items: center;
+}
+
+.form-control {
+  width: 95%;
+  height: 30px;
+  border-radius: 5px;
+  border: none;
+}
+
+.detail-link h4,
+.detail-link p {
+  align-self: flex-end;
+  color: #fff;
+  text-transform: uppercase;
```

Comment:
- [ ] KIndly not that the requirement says that your design should look like the template one provided, you need to user the same layout, icones, paddings, margins, font-size, composition and space between elements.... so that your home page design looks exactly the same as the template.

| Your design | > The template
---|--
![image](https://github.com/Abdelaziz-Mahdi/Most-Gainers/assets/65068771/4b3e2255-8d8c-492c-9e7b-75dfe00fe4f4) |  ![image](https://github.com/Abdelaziz-Mahdi/Most-Gainers/assets/65068771/8499f77d-d80f-46de-8a5e-7c653fbaefd2)

Reply:
Hello @andersonlebon ,
I hope you are fine, I read your review but I found the following
I don`t have the same icons and I don't work on the same subject that's why there are some differences and the requirements they specify there are no icons required:
Colors (select one main color for your website).
Typography: font face, size, and weight.
Layout: composition and space between elements.
Thanks.